data/migrate/errors-backfilling.csv

```diff
@@ -92,7 +92,7 @@ Error Code,Error Type,Message Template,Notes
 101,Error,No ozone data in #DAILY table
 102,Warning,#DAILY.Date found in non-chronological order
 103,Warning,#DAILY.Date has different year than #TIMESTAMP.Date
-104,Warning,Duplicate observations found in #DAILY table for Date={date}
+104,Warning,Duplicate observations found in #DAILY table for Date=(date)
```

Comment:
Those curly braces are important (used in `str.format` to substitute a value into the string), please change back.